File: lab15/5.py
def total_overs(balls):
    overs = balls//6
    bowled = balls % 6/10
    return overs+bowled


def main():
    balls = int(input("Enter number of balls: "))
    result = total_overs(balls)
    print(result)


if __name__ == "__main__":
    main()

File: PD/lab7/5.cpp
#include <iostream>
using namespace std;

bool isPrime(int number);

int main()
{
    int number;
    cout << "Enter Number: ";
    cin >> number;
    cout << isPrime(number);
}

bool isPrime(int number)
{
    int count = 0, result = 0;
    for (int i = 1; i <= number; i++)
    {
        if (number % i == 0)
        {
            count++;
        }
    }
    if (count == 2)
    {
        result = 1;
    }
    return result;
}